### PR TITLE
added bootstrap and event listeners for the modal color picker

### PR DESCRIPTION
**Pull request by MelanieRBaker**

### CODE REVIEW

**Comment by castle-c** [comment]
👯

File: dom.js [hunk 1]
```diff
@@ -39,5 +39,20 @@
     });
   }
 
+// event listeners for color picker
+  var saveButton = document.getElementById("saveBtn");
+    saveButton.addEventListener("click", function(){
```
Comment (aoswalt) [inline dom.js]: indentation

**Comment by aoswalt** [comment]
:+1:

**Comment by carter-harris** [comment]
👍

**Pull request merged**